Lib/test/test_typing.py

```diff
@@ -22,7 +22,6 @@
 from typing import is_typeddict
 from typing import no_type_check, no_type_check_decorator
 from typing import Type
-from typing import NewType
```

Comment:
This is not related, but it was unused and I felt like there's no need to create a separate PR for this. If required, I can revert this change.